_includes/meta.html

```diff
@@ -0,0 +1,8 @@
+<meta charset="utf-8" />
+<meta name="viewport" content="initial-scale=1,user-scalable=yes" />
+<meta name="description" content="Hierplane is an open-source Javascript library for visualizing and exploring trees." />
+<link rel="stylesheet" href="{% include url/url-base-path.html %}css/main.css" />
+<link rel="shortcut icon" href="{% include url/url-base-path.html %}favicon.ico" />
+<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Ubuntu:300,400,500" type="text/css" />
+<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Ubuntu+Mono" type="text/css" />
+<link rel="stylesheet" href="//unpkg.com/hierplane/dist/static/hierplane.min.css" type="text/css" />
```

Comment:
The styles aren't meta.  Can you make a separate file for those?

Also, there's a lot of fonts being loaded.  Have you ever tried the typography settings @jonesish documented here:
https://github.com/allenai/scholar/wiki/FE-Styleguide

They look great to me, and it means we don't need any custom fonts, which leads to a much faster font.  What do you think?

Also, none of these tags should be self closed, since it's HTML:

```
<meta /> -> <meta>
```

Reply:
Fixed meta self-closing and pulled css link files into a separate include. Will address the external font consideration as a follow-up.

Reply:
@codeviking As for the custom fonts, I'm going to make my case for why we should keep them:

1. **Aesthetics** - I think the Ubuntu font looks awesome. :) I think it makes the design better overall, as it looks fresh and more visually interesting. Helvetica Neue is a bit tired in my opinion. It got heavily overused in visual design over the past decade. You used to see it absolutely everywhere, and now you're seeing a lot more font diversity. Also, I used the same Ubuntu font for the logo type, so it helps tie the page together a bit.

2. **Cross-browser Consistency** - Responding to your suggestion, I'm not particularly keen on relying on system fonts unless they are cross-platform web standard fonts that are virtually guaranteed to be installed on every machine regardless of OS (e.g. Arial, Times, Verdana, Georgia, etc.). Using OS-specific system fonts (e.g. Helvetica Neue, Segoe UI, etc.) means that every user gets a different experience. In my opinion, that works against the goal of cross-browser consistency. Font weights are inconsistent as well. For example, the thin Helvetica Neue "Cutting through the clutter" heading on the Semantic Scholar homepage renders thick on Windows which gives it a totally different look. Also, sometimes one font manifestation vs another can cause text wrap at an inopportune time. With custom fonts, there is much more reliability in layout consistency. And of course, more brand consistency across platforms.

3. **Ubiquity** - As of this writing, custom web fonts are very common, especially among home pages for open-source code libraries. Here are some code library sites that use custom fonts: [Ubuntu](https://www.ubuntu.com/), [Linux](https://www.linux.org/), [TensorFlow](https://www.tensorflow.org/), [NodeJS](https://nodejs.org/en/), [Victory](http://formidable.com/open-source/victory/), [Kubernetes](https://kubernetes.io/), [Docker](https://www.docker.com/), [VueJS](https://vuejs.org/). Most of the custom fonts these sites are using are hosted by Google Fonts (though some are via Typekit or Font Awesome). Of these example sites, the average number of custom variants per site is about 7. Hierplane is currently using 4 (3 for Ubuntu and 1 for Ubuntu Mono, all hosted by Google Fonts). To be fair, there are some high-profile code library sites that do it the way you suggested. D3 and React were the notable ones, but IMHO I think the sites using custom fonts look better.

4. **Affordability** - First, the Google Fonts CDN seems to be fast and reliable. Also, my feeling is that the Hierplane site is so small at this point (and thankfully is being hosted by GitHub) that we can afford a couple of spit 'n polish nice-ities. And if we're trying to entice new users, then making our brochure pretty (see point 1) is going to help us more at this stage then in the future when Hierplane's fate has already been determined.

So there you have it. My case for why we should keep the fonts. I know from an engineering perspective fonts seem trivial, but from a design perspective, I believe there is significant value. That said, If you feel stronger about not using them than I do about keeping them then I will take them out. Let me know.

In the meantime, I've merged with a version that still uses the Ubuntu font, per your suggestion to go ahead and merge after a first pass on feedback.

Reply:
The google CDN is reliable, but it's less about latency in downloading each, rather it's that:

1. Each download is blocking.
2. It's fairly expensive to apply the downloaded font after it's loaded.

You should talk to @jonesish sometime about the research she's done behind her `font` size.  She figured out a combination of font families that achieves as much consistency as possible, but retains a super-fast experience -- I can tell you that it made a noticeable difference substituting it in for a single custom font used on the AI2 website (both in terms of load time and presentation).

I'll defer to you here, but from a web perf perspective it's not worth the tax imo.

https://www.keycdn.com/blog/web-font-performance/

Reply:
Thanks, @codeviking. I've added an issue for revisiting external font: https://github.com/allenai/hierplane/issues/71